OpenRA.Mods.Common/Activities/Air/HeliReturnToBase.cs

```diff
@@ -46,13 +47,27 @@ public override Activity Tick(Actor self)
 									.Where(a => a.Owner == self.Owner && rearmBuildings.Contains(a.Info.Name))
 									.ClosestTo(self);
 
-				if (nearestHpad == null)
+				if (nearestHpad == null) {
 					return ActivityUtils.SequenceActivities(new Turn(self, initialFacing), new HeliLand(self, true), NextActivity);
-				else
-					return ActivityUtils.SequenceActivities(new HeliFly(self, Target.FromActor(nearestHpad)));
+				} else {
+					var distanceFromHelipad = (nearestHpad.CenterPosition - self.CenterPosition).HorizontalLength;
+
+					//// if no pad is avaliable, move near one and wait
+					if (distanceFromHelipad > 6000) {
```

Comment:
It really should be an aircraft property? Is that this property is valid for just this Activity and, from my tests so far, I did not see any change needed for the different types of units that use this Activity. Maybe expose this constante when there is a need to customize the value?
The value looks kind of arbitrary indeed. With testing with multiple helicopters and only 2 helipads I saw that if they try to place they selfs 3 cells away from the helipad, they don't ram in to each other. 
I will change to 3072. But what do you think about exposing this constante? Do you think its the way to go?